src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -0,0 +1,2579 @@
+        </td>
+        </tr>
+    </tbody>
+    </table>
+
+    ### Examples
+
+    #### Record response time of a sleeping process [#record-stopwatch]
+
+    ```cs
+    Stopwatch stopWatch = Stopwatch.StartNew();
+    System.Threading.Thread.Sleep(5000);
+    stopWatch.Stop();
+    NewRelic.Api.Agent.NewRelic.RecordMetric("Custom/DEMO_Record_Metric", stopWatch.ElapsedMilliseconds);
+    ```
+
+    </Collapser>
+    <Collapser
+        id="RecordResponseTimeMetric"
+        title="RecordResponseTimeMetric"
+    >
+    ### Syntax
+
+    ```cs
+    NewRelic.Api.Agent.NewRelic.RecordResponseTimeMetric(string $metric_name, Int64 $metric_value)
+    ```
+
+    Records a custom metric with the given name and response time in milliseconds.
+
+    ### Requirements
+
+    * Compatible with all agent versions.
+    * Compatible with all app types.
+
+    ### Description
+
+    Records the response time in milliseconds for a [custom metric](/docs/agents/manage-apm-agents/agent-metrics/custom-metrics). To view these custom metrics, use the [query builder](/docs/query-your-data/explore-query-data/query-builder/use-advanced-nrql-mode-specify-data) to search metrics and create customizable charts. See also [`IncrementCounter()`](/docs/agents/net-agent/net-agent-api/incrementcounter-net-agent) and [`RecordMetric()`](/docs/agents/net-agent/net-agent-api/recordmetric-net-agent).
+
+    <Callout variant="important">
+    When creating a custom metric, start the name with `Custom/` (for example, `Custom/MyMetric`).
+    </Callout>
+
+    ### Parameters
+
+    <table>
+    <thead>
+        <tr>
+        <th width="25%">
+            Parameter
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            `$metric_name`
+
+            _string_
+        </td>
+
+        <td>
+            Required. The name of the response time metric to record. Only the first 255 characters are retained.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            `$metric_value`
+
+            _Int64_
+        </td>
+
+        <td>
+            Required. The response time to record in milliseconds.
+        </td>
+        </tr>
+    </tbody>
+    </table>
+
+    ### Examples
+
+    #### Record response time of a sleeping process [#record-stopwatch]
+
+    ```cs
+    Stopwatch stopWatch = Stopwatch.StartNew();
+    System.Threading.Thread.Sleep(5000);
+    stopWatch.Stop();
+    NewRelic.Api.Agent.NewRelic.RecordResponseTimeMetric("Custom/DEMO_Record_Response_Time_Metric", stopWatch.ElapsedMilliseconds);
+    ```
+
+    </Collapser>
+    <Collapser
+        id="SetApplicationName"
+        title="SetApplicationName"
+    >
+    ### Syntax
+
+    ```cs
+    NewRelic.Api.Agent.NewRelic.SetApplicationName(string $name[, string $name_2, string $name_3])
+    ```
+
+    Set the app name for data rollup.
+
+    ### Requirements
+
+    * Agent version 5.0.136.0 or higher.
```

Comment:
5.0.136.0 is EOL ( https://deploy-preview-17266--docs-website-netlify.netlify.app/docs/apm/agents/net-agent/getting-started/net-agent-eol-policy/ ), shouldn't this be all agent versions?